src/index.js

```diff
@@ -34,6 +34,14 @@ function bashEmulator (initialState) {
     run: function (input) {
       state.history.push(input)
 
+      // NOTE: we handle cat command without operand case here
```

Comment:
It's kind of ugly and doesn't work with `cat` aliases that we have: `less` and `more`